src/index.html

```diff
@@ -9,6 +9,93 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav__div">
+        
+        <img
+         class="image"
+         href="home"
+         src="./images/logo.png"
+         alt="moyo"
+        >
+      </div>
+      
+      <nav class="nav__lint">
+        <ul class="nav__item">
+          <li class="nav__li">
+            <a 
+            class="nav__link is-active"
+            href="#"
+            >
+            Apple
+            </a>
+          </li>
```

Comment:
That's an `item`

Correct formatting (check code style in the checklist):
```suggestion
          <li class="nav__li">
            <a 
              class="nav__link is-active"
              href="#"
            >
              Apple
            </a>
          </li>
```

![image](https://user-images.githubusercontent.com/86923089/216815407-731160bf-6e7f-48db-a50c-320a6f61528f.png)

Reply:
fix it everywhere